### DISCUSSION TITLE
我也搭了个ManifestAutoUpdate有需要可以自取

### DISCUSSION THREAD

**Discussion by BlankTMing** [pjy612/SteamManifestCache#27] [discussion]
[ManifestAutoUpdate](https://github.com/BlankTMing/ManifestAutoUpdate)
没多少游戏都是在网上爬的账号

**Comment by pjy612** [comment]
感谢分享
ps.70被你搬空了吧? 23333

**Reply by BlankTMing** [reply]
确实

**Reply by pjy612** [reply]
没办法，现在这种只能依赖账号。。。

**Reply by BlankTMing** [reply]
哎～网上这些账号都是低质量的，爬了4万个账号就4000个账号能用才5000+游戏网上的号被我爬了个精光了！实在找不到其他号了

**Reply by EricDasha** [reply]
好厉害0-0
